Makefile

```diff
@@ -25,7 +25,7 @@ ISO := $(or $(ISO), "") # ISO should point to a file that has the '.iso' extensi
 NUM_MASTERS :=  $(or $(NUM_MASTERS),3)
 WORKER_MEMORY ?= 8892
 MASTER_MEMORY ?= 16984
-NUM_WORKERS := $(or $(NUM_WORKERS),0)
+NUM_WORKERS := $(or $(NUM_WORKERS),2)
```

Comment:
lets change this envs default only for tests, not sure though how to do it in Makefie

Reply:
look now, changed only for the ``test`` target